src/Topshelf.Tests/CommandLine_Specs.cs

```diff
@@ -119,23 +119,39 @@ public void Should_create_an_install_host_with_description()
             Assert.AreEqual("Joe is good", installHost.Settings.Description);
         }
 
-        [Test]
-        public void Should_create_an_install_host_with_service_name_and_instance_name()
-        {
-            Host host = HostFactory.New(x =>
-                {
-                    x.Service<MyService>();
-                    x.ApplyCommandLine("install -servicename \"Joe\" -instance \"42\"");
-                });
-
-            Assert.IsInstanceOf<InstallHost>(host);
-            var installHost = (InstallHost)host;
-            Assert.AreEqual("Joe", installHost.Settings.Name);
-            Assert.AreEqual("42", installHost.Settings.InstanceName);
-            Assert.AreEqual("Joe$42", installHost.Settings.ServiceName);
-        }
-
-        [Test]
+		[Test]

```

Comment:
Whitespace should align, it looks like you're using 8 characters or tabs. Should be using 4 spaces per indent.